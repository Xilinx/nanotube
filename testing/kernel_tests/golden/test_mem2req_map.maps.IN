###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
nanotube_map: 0 0 3 38
key: 66 6f 6f
value: 01 aa
       02 00 fe ca
       04 00 00 00 cc ba ed fe
       08 00 00 00 00 00 00 00 11 22 33 44 55 66 77 88
       ff ee dd cc bb aa 99 88
end

