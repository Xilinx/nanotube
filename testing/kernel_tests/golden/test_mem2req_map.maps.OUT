###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
# Testing kernel map_read_test
# Maps after packet 0
nanotube_map: 0 0 3 38
key: 66 6f 6f
value: 01 aa 02 00 fe ca 04 00 00 00 cc ba ed fe 08 00
       00 00 00 00 00 00 11 22 33 44 55 66 77 88 ff ee
       dd cc bb aa 99 88
end

# Testing kernel map_write_test
# Maps after packet 0
nanotube_map: 0 0 3 38
key: 66 6f 6f
value: 01 bb 02 00 0f f0 04 00 00 00 0c b0 fe ca 08 00
       00 00 00 00 00 00 ef cd ab 89 67 45 23 01 ff ee
       dd cc bb aa 99 88
end

# Testing kernel map_read_test2
# Maps after packet 0
nanotube_map: 0 0 3 38
key: 66 6f 6f
value: 01 bb 02 00 0f f0 04 00 00 00 0c b0 fe ca 08 00
       00 00 00 00 00 00 ef cd ab 89 67 45 23 01 ff ee
       dd cc bb aa 99 88
end

# Testing kernel map_memset_memcpy_test
# Maps after packet 0
nanotube_map: 0 0 3 38
key: 66 6f 6f
value: 01 bb 02 00 0f f0 04 00 00 00 0c b0 fe ca 08 00
       00 00 00 00 00 00 ef cd ab 89 67 45 23 01 ff ee
       dd cc bb aa 99 88
end

